FALSEHOODS.md

```diff
@@ -118,3 +118,18 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  A given phone number has only one associated phone line. (Corollary: if a
+    call is ongoing with a given phone number, another call cannot be made to
+    or from that number until the first call finishes.)
+
+    In addition to services like call waiting, many phone services (especially
+    for businesses) allow multiple outbound calls from the same phone number.
+
+1.  The area code or country code of a phone number gives some idea of the
+    user's location, time zone, or preferred language.
```

Comment:
I think it is actually fair it gives you some idea. If they have an American phone number, statistically speaking, it's more likely that they are in the US than somewhere else, and that they speak English.

I get your point though that it shouldn't be assumed that it is accurate.

Would you be happier with this more definite sentence instead: "The country or area code of a phone number indicates the user's location, time-zone or preferred language."

We could also add "place of residence" to the list.